javagen/src/main/java/com/azure/autorest/model/clientmodel/ClassType.java

```diff
@@ -16,87 +16,315 @@
-    public static final ClassType AzureEnvironment = new ClassType.Builder().packageName("com.azure.core.management").name("AzureEnvironment").build();
-    public static final ClassType HttpClient = new ClassType.Builder().knownClass(com.azure.core.http.HttpClient.class).build();
-    public static final ClassType AndroidHttpClient = new ClassType.Builder().packageName("com.azure.android.core.http").name("HttpClient").build();
-    public static final ClassType HttpPipelinePolicy = new ClassType.Builder().knownClass(com.azure.core.http.policy.HttpPipelinePolicy.class).build();
-    public static final ClassType AndroidHttpPipelinePolicy = new ClassType.Builder().packageName("com.azure.android.core.http").name("HttpPipelinePolicy").build();
-    public static final ClassType HttpLogOptions = new ClassType.Builder().knownClass(com.azure.core.http.policy.HttpLogOptions.class).build();
-    public static final ClassType AndroidHttpLogOptions = new ClassType.Builder().packageName("com.azure.android.core.http.policy").name("HttpLogOptions").build();
-    public static final ClassType Configuration = new ClassType.Builder().knownClass(com.azure.core.util.Configuration.class).build();
-    public static final ClassType ServiceVersion = new ClassType.Builder().knownClass(com.azure.core.util.ServiceVersion.class).build();
-    public static final ClassType AzureKeyCredential = new ClassType.Builder().knownClass(com.azure.core.credential.AzureKeyCredential.class).build();
-    public static final ClassType RetryPolicy = new ClassType.Builder().knownClass(com.azure.core.http.policy.RetryPolicy.class).build();
-    public static final ClassType RetryOptions = new ClassType.Builder().knownClass(com.azure.core.http.policy.RetryOptions.class).build();
-    public static final ClassType AndroidRetryPolicy = new ClassType.Builder().packageName("com.azure.android.core.http.policy").name("RetryPolicy").build();
-    public static final ClassType JsonPatchDocument = new ClassType.Builder().knownClass(com.azure.core.models.JsonPatchDocument.class).build();
-    public static final ClassType BinaryData = new ClassType.Builder().knownClass(com.azure.core.util.BinaryData.class).defaultValueExpressionConverter((String defaultValueExpression) -> java.lang.String.format("BinaryData.fromObject(\"%s\")", defaultValueExpression)).build();
-    public static final ClassType RequestOptions = new Builder().packageName("com.azure.core.http.rest").name("RequestOptions").build();
-    public static final ClassType ClientOptions = new Builder().knownClass(com.azure.core.util.ClientOptions.class).build();
-    public static final ClassType HttpRequest = new Builder().knownClass(com.azure.core.http.HttpRequest.class).build();
-    public static final ClassType HttpHeaders = new Builder().knownClass(com.azure.core.http.HttpHeaders.class).build();
+    public static final ClassType Void = new ClassType.Builder(false).knownClass(Void.class).build();
```

Comment:
If there are no changes to this class, we should revert to the previous formatting.